settings.gradle

```
@@ -188,6 +188,7 @@ include ':instrumentation:spring:starters:otlp-exporter-starter'
 include ':instrumentation:spring:starters:zipkin-exporter-starter'
 include ':instrumentation:spymemcached-2.12'
 include ':instrumentation:twilio-6.6'
+include ':instrumentation:struts-2'
```

also, can you send a separate PR adding struts to the main README? (separate PR so we can wait to update the README when we release the next version)

```suggestion
include ':instrumentation:struts-2.0'
```

---

Updated version to 2.3. Main README PR: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1636